python/lsst/ip/isr/photodiode.py

```diff
@@ -297,10 +297,17 @@ def integrateChargeSum(self):
         # The current per interval to use for baseline subtraction
         # without assuming all of the dt values are the same:
         current = charge/dt
-        # For the baseline current level, select current values < 5%
-        # of the maximum, measured relative to the overall minimum.
+        # To determine the baseline current level, exclude points with
+        # signal levels > 5% of the maximum (measured relative to the
+        # overall minimum), and extend that selection 2 entries on
+        # either side to avoid otherwise low-valued points that sample
+        # the signal ramp and which should not be included in the
+        # baseline estimate.
         dy = np.max(current) - np.min(current)
-        index = np.where(current < dy/20. + np.min(current))
-        bg_current = np.sum(charge[index])/np.sum(dt[index])
+        signal = np.where(current > dy/20. + np.min(current))
```

Comment:
If this is a 1d array, you can do `signal, = np.where(current > dy/20. + np.min(current))` and just get the indices and below you can do `signal[0]` and `signal[-1]`.  And although this is what I still do, the numpy recommendation is instead to do `signal, = (current > dy/20. + np.min(current)).nonzero()`.